Match with the latest math_private restructuring in glibc

as done here

http://sourceware.org/ml/libc-alpha/2012-03/msg00302.html

Signed-off-by: Khem Raj <raj.khem@gmail.com>

Upstream-Status: Pending

Index: libc/ports/sysdeps/powerpc/powerpc32/e500/fpu/math_private.h
===================================================================
--- libc.orig/ports/sysdeps/powerpc/powerpc32/e500/fpu/math_private.h	2012-07-17 22:58:20.340405923 -0700
+++ libc/ports/sysdeps/powerpc/powerpc32/e500/fpu/math_private.h	2012-07-17 22:58:21.120405923 -0700
@@ -1 +1 @@
-#include <math/math_private.h>
+#include_next <sysdeps/generic/math_private.h>
Index: libc/ports/sysdeps/powerpc/powerpc32/e500/fpu/fraiseexcpt.c
===================================================================
--- libc.orig/ports/sysdeps/powerpc/powerpc32/e500/fpu/fraiseexcpt.c	2012-07-17 23:15:15.872405939 -0700
+++ libc/ports/sysdeps/powerpc/powerpc32/e500/fpu/fraiseexcpt.c	2012-07-17 23:18:42.760405602 -0700
@@ -23,7 +23,7 @@
 
 #undef feraiseexcept
 
-#define __FERAISEEXCEPT_INTERNAL feraiseexcept
+#define __FERAISEEXCEPT_INTERNAL __feraiseexcept
 #include "../spe-raise.c"
 
-libm_hidden_def (feraiseexcept)
+libm_hidden_ver (__feraiseexcept, feraiseexcept)
